docs/UserGuide.md

```diff
@@ -143,19 +150,21 @@ Here are some **key points** to keep in mind when using commands:
 - You can add fields in **any order**. <br>
   _**Example:** If the command specifies `n/NAME p/PHONE_NUMBER`, then `p/PHONE_NUMBER n/NAME` is also acceptable._
 
-<box type="tip" theme="success">
+<box type="tip" theme="success" light seamless style="border: 1px solid">
```

Comment:
Might be better to remove seamless and the style arguments here, since they get overwritten. It should probably be something like

<box type="tip" theme="success" light>